internal/jobs/launch_instance_azure.go

```diff
@@ -72,6 +73,13 @@ func HandleLaunchInstanceAzure(ctx context.Context, job *worker.Job) {
 		logger.Debug().Msg("Resource group has not been set, defaulting to 'redhat-deployed'")
 		args.ResourceGroupName = DefaultAzureResourceGroupName
 	}
+	if args.Location != "" {
+		// Match for location - availability zone - suffix and remove it
+		res, e := regexp.MatchString(`_[1-6]\z`, args.Location)
+		if e == nil && res {
+			args.Location = args.Location[0 : len(args.Location)-2]
+		}
+	}
```

Comment:
Can you explain this?

Reply:
I want to strip the availability zone suffix here, to keep supporting passing this as we forced that on users in the past unfortunatelly, so some clients migh have that already (`eastus_1`, etc)... I don't want to break those clients.

Basically this: https://go.dev/play/p/V6nK3LY4ajN